gvsbuild/projects/libzip.py

```diff
@@ -26,8 +26,8 @@ def __init__(self):
         Project.__init__(
             self,
             "libzip",
-            archive_url="https://nih.at/libzip/libzip-1.8.0.tar.gz",
-            hash="30ee55868c0a698d3c600492f2bea4eb62c53849bcf696d21af5eb65f3f3839e",
+            archive_url="https://nih.at/libzip/libzip-1.9.0.tar.gz",
```

Comment:
I know it was like that even before, but any reason why we are not using the official domain? https://libzip.org/download/

Reply:
Good idea @pbor!